src/osp/Active/ActiveScene.h

```diff
@@ -306,6 +307,9 @@ struct ACompCamera
     Matrix4 m_projection;
     Matrix4 m_inverse;
 
+    // If empty, GL::DefaultFramebuffer is used
+    DependRes<Magnum::GL::Framebuffer> m_renderTarget;
```

Comment:
It's a little hard to reason about this yet because we only have 1 camera per scene, but if you have e.g. one outboard camera on a ship and then the main player camera inside, you need the first camera to render to an offscreen framebuffer which is then used as a texture in the cockpit scene when the players camera renders the cockpit displays to the main framebuffer.

Reply:
The camera doesn't render to an instrument screen, it's the instrument screen that looks for a camera to render from. Multiple instrument screens should be able to look through the same camera as well.

Reply:
That makes sense, as long as the interface is abstracted away from the rendering stuff. My (currently WIP) revamped map screen/solar system code is currently providing me with yet another datapoint in terms of rendering schemes and required data, so when I get a chance I'll revisit this PR and see what I can come up with.